### DISCUSSION TITLE
Resume Bundle X ATS Scoring

### DISCUSSION THREAD

**Discussion by sakshiigdtuw** [recodehive/resume-pitch#96] [Q&A]
Name: Sakshi Chaudhary
 Linkedin Link: https://www.linkedin.com/in/sakshi-chaudhary-174770289/
 Resume Link: [Sakshi_CV (5).pdf](https://github.com/user-attachments/files/17009355/Sakshi_CV.5.pdf)
 GitHub Link: https://github.com/sakshiigdtuw
 I am Second year student pursuing B.Tech (ECE with AI specialization) at IGDTUW(Indira Gandhi Delhi Technical University) looking for internships in software engineer role.

**Comment by sanjay-kv** [comment]
I didnt see any topmate request for this 
could you check that again: 
if you wanna book again: https://topmate.io/sanjaykv/1047284/pay
use the same freecode you used before.
